libs/ui/src/export_logs_modal.test.tsx

```diff
@@ -30,6 +32,18 @@ const fileSystemEntry: KioskBrowser.FileSystemEntry = {
   mtime: new Date(),
 };
 
+const systemAdministratorAuthStatus: DippedSmartCardAuth.SystemAdministratorLoggedIn =
+  {
+    status: 'logged_in',
+    user: fakeSystemAdministratorUser(),
+    programmableCard: { status: 'no_card' },
+  };
```

Comment:
I ultimately didn't replace the old `Dipped.fakeSystemAdministratorAuth()`, etc. utils with analogs for the new auth types. I think the old auth types really benefited from the helpers because of all of the functions on the types, e.g. `systemAdministratorAuth.card.programUser`. The new auth types are simple enough to construct as needed. Given some of our new API mocking patterns, there are fewer points where we need to construct mock auth statuses anyway (usually just in the API mocks themselves)

Reply:
Sounds good to me!